charts/datadog/templates/_kubernetes_state_core_config.yaml

```diff
@@ -29,6 +29,7 @@ kubernetes_state_core.yaml.default: |-
       - poddisruptionbudgets
       - storageclasses
       - volumeattachments
+      - verticalpodautoscalers
```

Comment:
base on your PR's comment. I think you can solve the issue of not enabling the VPA collection by adding an option like for [secret](https://github.com/DataDog/helm-charts/blob/76f1416871df57da834e420f7b933ac7d3aedbf1/charts/datadog/templates/_kubernetes_state_core_config.yaml#L9-L11)

Reply:
Thought that would be a good solution too. Added that in, also adjusted the RBAC template to only give that corresponding permission for the `verticalpodautoscalers` when enabled.